windows/rngallery/Package.appxmanifest

```diff
@@ -13,8 +13,8 @@
 
  <Identity
     Name="Microsoft.ReactNativeGallery"
-    Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US"
-    Version="1.0.18.0" />
+    Publisher="CN=Microsoft Corporation (Internal Use Only), O=Microsoft Corporation, L=Redmond, S=Washington, C=US"

```

Comment:
I don't think we want to merge this change into the repo, only needed as a local change when signing a new package through the process you used.